### PR TITLE
feat/ add register class

### PR DESCRIPTION
**Pull request by luszczyk597**
Please review

### CODE REVIEW

File: src/Register.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import java.util.Scanner;
+
+public class Register {
+    public static void register() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.printf("Podaj nazwe:");
```
Comment (Avikele) [inline src/Register.java:6]: English pls

File: src/Register.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        String username = scanner.nextLine();
+        System.out.printf("Podaj email:");
+        String email = scanner.nextLine();
+        System.out.printf("Podaj hasło:");
```
Comment (Avikele) [inline src/Register.java:10]: Po angielsku

**Comment by luszczyk597** [comment]
fixed

**Review by Avikele — APPROVED** [review]
lgtm

**Pull request merged**